source/options-storage.ts

```diff
@@ -56,12 +56,12 @@ export default getOptions(location.protocol.startsWith('http') ? location.host :
 
 export async function getAllOptions(): Promise<Map<string, OptionsSync<RGHOptions>>> {
 	const optionsByDomain = new Map<string, OptionsSync<RGHOptions>>();
-	optionsByDomain.set('github.com', getOptions('github.com'));
+	optionsByDomain.set('https://github.com', getOptions('github.com'));
 
 	const {origins} = await getAdditionalPermissions();
 	for (const origin of origins) {
-		const {host} = new URL(origin);
-		optionsByDomain.set(host, getOptions(host));
+		const {protocol, host} = new URL(origin);
+		optionsByDomain.set(`${protocol}//${host}`, getOptions(host));
```

Comment:
We don’t need to consider protocol changes. GitHub should only be available on https